app/views/customers/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Customer Details</h1>
+
+<div id = "customer-name">
+    <p>Customer Name: <%= @customers.name %></p>
+</div>
+<div id = "item-list">
+    <ul>
+        <% @customers.items.each do |item|%>
+            <li><%= item.name %> - Price: <%= item.price %> - Supermarket: <%= item.supermarket.name %></li>
+        <% end %>
+    </ul>
+</div>
+
+<h2>Add Item to Customer</h2>
+<%= form_with url: "/customers/#{@customers.id}", method: :post, data: {turbo: false} do |form|%>
+    <div id = "form-item">
+        <%= form.label :item_id %>
+        <p><%= form.text_field :item_id %></p>
+        <%= form.submit "Add Item"%>
+    </div>    
```

Comment:
Syntax on the form looks good. I'm not sure why only line 18 is wrapped in a <p> tag though.